addons/edi_stock/data/edi_move_cancellation_data.xml

```diff
@@ -57,13 +57,9 @@
                     <div style="display:table-row;">
                         <div style="display:table-cell; padding: 0px 5px;">${move.product_id.name_get()[0][1]}</div>
                         <div style="display:table-cell; padding: 0px 5px;">
-                            % if move.sale_line_id:
-                                ${move.sale_line_id.product_uom_qty}
```

Comment:
Q: Why have we removed this, is it just to simplify?

Reply:
This was removed as this assigns the quantity ordered rather than cancelled.

Reply:
We might want to check the original story / sign off, as i'm pretty sure product requested this functionality originally?

Reply:
I think these should be the same thing for most cases. Not sure how the old code would work if you had back orders and you cancelled a back order, but I think your change would only show the qty cancelled in the backorder, which is correct.

Reply:
oh wait, that was basically the issue in the story :smiling_face_with_tear: I will resolve